### PR TITLE
[docs-only] Add documemtation for the graph users and group APIs

### PR DESCRIPTION
**Pull request by rhafer**

### CODE REVIEW

**Review by micbar — APPROVED** [review]
Awesome!

File: docs/extensions/graph/groups.md [hunk 1]
```diff
@@ -0,0 +1,199 @@
...
+```
+curl -k --request DELETE 'https://localhost:9200/graph/v1.0/groups/7a20f238-8a22-4458-902d-47674c317e5f' -u user:password
+```
+
+When successful the API returns no Response Body and the HTTP status code 204 (No Content)
+
+#### `PATCH /groups/{id}`
+
+Updating attributes of a single group is supposed to be done with a patch request. This is however currently not fully 
+implemented for our write-enabled backends. The PATCH request can however to used to add multiple members to a group at once.
```
Comment (kobergj) [inline docs/extensions/graph/groups.md:150]: typo
```suggestion
implemented for our write-enabled backends. The PATCH request can however be used to add multiple members to a group at once.
```

File: docs/extensions/graph/groups.md [hunk 2]
```diff
@@ -0,0 +1,199 @@
...
+
+#### `DELETE /groups/{id}`
+
+Example:
+
+```
+curl -k --request DELETE 'https://localhost:9200/graph/v1.0/groups/7a20f238-8a22-4458-902d-47674c317e5f' -u user:password
+```
+
+When successful the API returns no Response Body and the HTTP status code 204 (No Content)
```
Comment (kobergj) [inline docs/extensions/graph/groups.md:145]: `Response Body` should always be lower case as it is not pointing to a resource

File: docs/extensions/graph/groups.md [hunk 3]
```diff
@@ -0,0 +1,199 @@
...
+	"https://localhost:9200/graph/v1.0/users/c54b0588-7157-4521-bb52-c1c8ca84ea71"
+    ]
+}
+```
+
+When successful the API returns no Response Body and the HTTP status code 204 (No Content)
+
+### Removing a member
+
+#### `DELETE /groups/{groupid}/members/{id}/$ret`
```
Comment (kobergj) [inline docs/extensions/graph/groups.md:189]: typo?
```suggestion
#### `DELETE /groups/{groupid}/members/{id}/$ref`
```

**Review by kobergj — CHANGES_REQUESTED** [review]

**Comment by rhafer** [comment]
@kobergj seems I was a little fast with merging this. I've created #3227 to address your feedback. Thanks!

**Pull request merged**